library/common-library/templates/_verbose.tpl

```diff
@@ -0,0 +1,26 @@
+{{- /*
+Abstraction of the verbose toggle.
+This helper allows to override the global `.global.verbose` with the value of `.verbose`.
+Returns "true" if `verbose` is enabled, otherwise "" (empty string)
+*/ -}}
+{{- define "common.verbose" -}}
+{{- /* `get` will return "" (empty string) if value is not found, and the value otherwise, so we can type-assert with kindIs */ -}}
+{{- if (get .Values "verbose" | kindIs "bool") -}}
+    {{- if .Values.verbose -}}
```

Comment:
In newerelic-infrastructure | -v3 the value is called verboseLog, should we treat it here or have a compatibility layer helper in that repo?

Reply:
You are right. Solved in [#3c553eb](https://github.com/newrelic/helm-charts/pull/752/commits/3c553ebce4127a9f81e6e4c39fd5344df8a584b4)